examples/simple/cereal.csv

```diff
@@ -0,0 +1,78 @@
+name,mfr,type,calories,protein,fat,sodium,fiber,carbo,sugars,potass,vitamins,shelf,weight,cups,rating

```

Comment:
Please convert these CRLF endings to LF.

Reply:
I don't know what this means.

Reply:
The file was obtained from kaggle, and is provided exactly as it was downloaded. I'm not sure there is good reason to change it?

Reply:
I prefer not have CRLF line endings stored in the repo.  End users can of course customize git to check them out to match their preference.  I've squashed your fixup commits, and added this change in.